src/utilities/theme/role-variants.ts

```diff
@@ -148,7 +148,7 @@ export const roleVariants: RoleVariants = {
       name: 'borderDisabled',
       description: 'For use as a an interactive outline on disabled elements.',
       light: {lightness: 95},
-      dark: {lightness: 70},
+      dark: {lightness: 30},
```

Comment:
Adjusting dark values for `borderDisabled` and `shadowFromDirectLight` to better visually match the light values

Reply:
Quite a few of the dark values are untested and need tweaking, thanks for this.